src/lib.rs

```diff
@@ -283,6 +285,21 @@ impl Check {
         self.log_level.as_ref()
     }
 
+    /// Set the termcolor [color choice](termcolor::ColorChoice)
+    /// If `None`, the use of colors will be determined automatically by
+    /// the `CARGO_TERM_COLOR` env var and tty type of output
```

Comment:
Just ensuring the sentences don't run into each other.
```suggestion
    /// Set the termcolor [color choice](termcolor::ColorChoice).
    /// If `None`, the use of colors will be determined automatically by
    /// the `CARGO_TERM_COLOR` env var and tty type of output.
```